README.md

```diff
@@ -2,7 +2,191 @@
+  on work by Hofstadter.
+* "demo0.mm" - a simple formal system used as a demonstration in
+  Chapter 2 of the Metamath book.
+
+## How are they verified?
+
+Changes ("commits") to any database are first automatically verified
+before they are accepted, using GitHub actions.
+
+The set.mm and iset.mm databases are the ones primarily being updated.
+Every change to these databases are verified by the following
+different verifiers:
+
+* metamath.exe aka Cmetamath (the original C verifier by Norman Megill)
+* checkmm (a C++ verifier by Eric Schmidt)
+* smetamath-rs (smm3) (a Rust verifier by Stefan O'Rear)
+* mmj2 (a Java verifier by Mel L. O'Cat and Mario Carneiro)
+* mmverify.py (a Python verifier by Raph Levien)
+
+Note that these are different verifiers written in different programming
+languages by different people. In addition, the verification algorithm
+is intentionally simple (it fits in two pages in the Matamath book).
+This provides *extremely* high confidence that the proofs are correct.
+Most math proofs are not formally verified at all (that is, where a
+computer verification verifies every step). The rest are typically only
+verified by a single program (which might have an error in it).
+These multiple independent verifications provide very high confidence
+of their being correct.
+
+All other databases are verified by at least one verifier.
+
+## How can I contribute?
+
+First, take a look at some existing proofs.
+Walking through the 
+[Metamath Proof Explorer (MPE, aka set.mm)](http://us.metamath.org/mpegif/mmset.html) is a good way to start.
+
+To create a proof, you need to pick a tool.
+The Metamath system specifies a common file format, not a single tool
+everyone has to use.
+Many in the community use the [mmj2 tool](http://us.metamath.org#book).
+The video [Introduction to Metamath and mmj2](https://www.youtube.com/watch?v=Rst2hZpWUbU) demonstrates using mmj2.
+The mmj2 tool includes an interactive tutorial, and if you just want to watch,
+there's a video giving a
+[Walkthrough of the tutorial in mmj2](https://www.youtube.com/watch?v=87mnU1ckbI0&t=2094s).
+You should start small and try to reprove what's already
+proven with Metamath before you try to prove something new.
+If you're contributing to set.mm or iset.mm, take a look at its
+[conventions and style](http://us.metamath.org/mpegif/conventions.html).
+
+At some point you should join the
+[Metamath mailing list](https://groups.google.com/g/metamath).
+We're a friendly community and we would love to have more collaborators!
+
+The Wiki page
+[Getting started with contributing](https://github.com/metamath/set.mm/wiki/Getting-started-with-contributing) will walk you the process of contributing
+a final proof once you have one.
+Basically, you'll create a "pull request" on GitHub to request
+merging of some change(s).
+**Warning**: Changes are made to the **develop** branch.
+
+## What are the requirements for acceptance (merging)?
```

Comment:
I think this section should be moved to the https://github.com/metamath/set.mm/blob/develop/CONTRIBUTING.md file.

The README should say it is great to contribute and something about how to write proofs and the like, but the details of pull requests and that sort of thing belong in CONTRIBUTING if I correctly understand the usual github conventions.

Reply:
That is the usual convention. These instructions are pretty short, so I thought I could get away with just slipping them into the README. Happy to move them to a CONTRIBUTING.md file, though - but if I do, I'll create a link to it from the README.

Reply:
Done.